python/lsst/ts/MTAlignment/component.py

```diff
@@ -149,15 +150,15 @@ def __init__(self):
                 if randrange(5) == 4:
                     cam_aligned = True
                     print("cam aligned")
-            asyncio.sleep(0.5)
+            await asyncio.sleep(0.5)
             if m2_aligned == False:
                 await self.measure_m2()
                 m2_offset = await self.query_m2()
                 if randrange(5) == 4:
                     m2_aligned = True
                     print ("M2 aligned")
             n -= 1
-            asyncio.sleep(0.5)
+            await asyncio.sleep(0.5)
```

Comment:
Why to you need to sleep for 0.5 seconds here?

Reply:
I think I put these in to make communication with the vendor's code work. I will need to check tomorrow to see if this is still necessary as I have a newer version of it now.